formatter.js

```diff
@@ -0,0 +1,54 @@
+/*
+  Custom eslint formatter for eslint-plugin-i18n-json to allow better error message display.
+  Heavily inspired from https://github.com/sindresorhus/eslint-formatter-pretty.
+*/
+
+const chalk = require('chalk');
+const plur = require('plur');
+const logSymbols = require('log-symbols');
+const indentString = require('indent-string');
+const path = require('path');
+
+const CWD = process.cwd();
+
+const formatter = (results) => {
+  const total = {
```

Comment:
can we find a better name for it (that describes what totals are we talking about)?